tests/issues/issue-635.php

```diff
@@ -30179,9 +30179,9 @@
             'Device_Pointing_Method' => 'touchscreen',
             'Device_Code_Name' => 'A680',
             'Device_Brand_Name' => 'Lenovo',
-            'RenderingEngine_Name' => 'Blink',
```

Comment:
This one should actually be WebKit because it's earlier than Chrome 27.  Previous to this PR, the `1.20` version was in the same file as the `13.9`+ patterns, which are on Blink.  In my changes above, this (seemingly android only) version was moved to a file that contains earlier versions of the browser, which all appear to be previous to Chrome 27.